packages/react-tree-grid/.storybook/tsconfig.json

```diff
@@ -22,6 +22,7 @@
     "../stories/**/*.stories.tsx",
     "../stories/**/*.stories.mdx",
     "*.ts",
-    "*.js"
+    "*.js",
+    "../stories/VariantB.stories.tsx"
```

Comment:
I see my comment caused confusion :D sorry about that...

you initial change was correct. that "whoops" was my surprise that you exposed invalid setup , please revert this change to your former change. ty